pkg/aggregator/aggregator.go

```diff
@@ -274,7 +274,10 @@ func (agg *BufferedAggregator) AddAgentStartupTelemetry(agentVersion string) {
 		SampleRate: 1,
 		Timestamp:  0,
 	}
-	agg.eventIn <- event
+	if agg.hostname != "" {
```

Comment:
you can create the `event` only if the condition is true

Reply:
yes it's the starup event to inform that a Datadog Agent has started on a specific host, its format on the events page is `<hostname>'s Datadog Agent has started`.
For Fargate we set the hostname to `""` which result having `None’s datadog agent started` as an event

Reply:
do you suggest doing that way ?

```golang
if agg.hostname != "" {
    event := metrics.Event{
		Text:           fmt.Sprintf("Version %s", agentVersion),
		SourceTypeName: "System",
		Host:           agg.hostname,
		EventType:      "Agent Startup",
	}
    agg.eventIn <- event
}
```

Reply:
yes or can be also

```golang
if agg.hostname != "" {
    agg.eventIn <- metrics.Event{
		Text:           fmt.Sprintf("Version %s", agentVersion),
		SourceTypeName: "System",
		Host:           agg.hostname,
		EventType:      "Agent Startup",
	}
}

```

Reply:
thank you! I've made the change